src/hotspot/cpu/x86/x86_64.ad

```diff
@@ -1758,6 +1764,19 @@ bool Matcher::is_spillable_arg(int reg)
   return can_be_java_arg(reg);
 }
 
+uint Matcher::int_pressure_limit()
+{
+  return (INTPRESSURE == -1) ? _INT_REG_mask.Size() : INTPRESSURE;
+}
+
+uint Matcher::float_pressure_limit()
+{
+  // The following default threshold works best for LCM's register pressure
+  // scheduling on x64.
+  uint default_float_pressure_threshold = _FLOAT_REG_mask.Size() - 2;
```

Comment:
Why it works best? Did you tested different values or it is required for correctness? If for correctness - why?

Reply:
Not for correctness.
The value was tested and verified in SPECjvm2008 by the author of LCM's register pressure scheduling.
Therefore I continue to use the original value.
“I did in fact experiment around with different values for both float and int pressure thresholds.  The versions in changelist work best for x64 and x86.” [1]
[1] http://mail.openjdk.java.net/pipermail/hotspot-compiler-dev/2015-September/019005.html

Reply:
Okay, I got it. So originally for 64-bit FLOATPRESSURE was 14 (16 -2) and that is where `-2` came from.

Reply:
But for avx512 it was 28 before and it is 30 now. I don't know which number is better for performance. But your change made difference.

Reply:
Thanks. I update accordingly.